### ISSUE TITLE
Test for text resizing errors when bbc-a11y is updated

### ISSUE THREAD

**Issue by radiocontrolled** [bbc/simorgh#418]
**Describe the bug**
The following [bbc-a11y](https://github.com/bbc/bbc-a11y) test, `Text must be styled with units that are resizable in all browsers`, is skipped in Simorgh (see: https://github.com/bbc/simorgh/blob/latest/a11y.js#L3). 

This is because  bbc-a11y itself always grabs the computed size of text in pixels, meaning the test will always fail. There is an issue raised on bbc-a11y related to this here: https://github.com/bbc/bbc-a11y/issues/262#issuecomment-409267398

**Additional context**
When this issue on bbc-a11y is fixed, we should adapt Simorgh so that it is not skipping `Text must be styled with units that are resizable in all browsers`, and fix any bugs that arise.

**Comment by greenc05** [comment]
Just checking should this be closed @radiocontrolled @jtart?

**Comment by jtart** [comment]
My mistake, I misread the title!

There's an open PR to fix this in bbc-a11y now, https://github.com/bbc/bbc-a11y/pull/266.

**Comment by greenc05** [comment]
No problem, thanks @jtart. Please could you and/or @radiocontrolled review bbc/bbc-a11y#266 then this can be unblocked

**Comment by radiocontrolled** [comment]
Closed by https://github.com/bbc/simorgh/issues/497
